src/stapi_fastapi/routers/root_router.py

```diff
@@ -104,45 +130,78 @@ def __init__(
             tags=["Orders"],
         )
 
+        if ASYNC_OPPORTUNITIES in conformances:
+            self.add_api_route(
+                "/searches/opportunities",
+                self.get_opportunity_search_records,
+                methods=["GET"],
+                name=f"{self.name}:list-opportunity-search-records",
+                summary="List all Opportunity Search Records",
+                tags=["Opportunities"],
+            )
+
+            self.add_api_route(
+                "/searches/opportunities/{search_record_id}",
+                self.get_opportunity_search_record,
+                methods=["GET"],
+                name=f"{self.name}:get-opportunity-search-record",
+                summary="Get an Opportunity Search Record by ID",
+                tags=["Opportunities"],
+            )
+
     def get_root(self, request: Request) -> RootResponse:
-        return RootResponse(
-            id="STAPI API",
-            conformsTo=self.conformances,
-            links=[
-                Link(
-                    href=str(request.url_for(f"{self.name}:root")),
-                    rel="self",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:conformance")),
-                    rel="conformance",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:list-products")),
-                    rel="products",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:list-orders")),
-                    rel="orders",
-                    type=TYPE_JSON,
-                ),
+        links = [
+            Link(
+                href=str(request.url_for(f"{self.name}:root")),
+                rel="self",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:conformance")),
+                rel="conformance",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:list-products")),
+                rel="products",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:list-orders")),
+                rel="orders",
+                type=TYPE_GEOJSON,
+            ),
+            Link(
+                href=str(request.url_for(self.openapi_endpoint_name)),
+                rel="service-description",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(self.docs_endpoint_name)),
+                rel="service-docs",
+                type="text/html",
+            ),
+        ]
+
+        if self.supports_async_opportunity_search:
+            links.insert(
```

Comment:
overall, I don't think it matters where they are in the list. But, instead of inserting them not at the end, I think you should move the openapi & docs endpoints to right after self, and just insert this at the end.